eisbuk-admin/cypress/support/commands.js

```diff
@@ -36,28 +37,41 @@ connectFunctionsEmulator(functions, "localhost", 5001);
 
 /**
  * Set up app for testing and log in as default admin
+ * @param {boolean} doLogin - whether to log in as default admin
  */
-Cypress.Commands.add("initAdminApp", () => {
+Cypress.Commands.add("initAdminApp", async (doLogin = true) => {
   // create a random organization name in order to run each test
   // against it's own organization, using the same db without conflicts
   const organization = uuidv4();
+  console.log(doLogin);
+  debugger;
```

Comment:
I would either remove the `console.log` (if it was a WIP debug check), or make it more verbose if it's part of standard reporting. I don't understand what the debugger does here, to me it seems as just an uncalled expression.

Reply:
This is what code reviews are for! I forgot to take these out.